### PR TITLE
Add timeout component for authentication pages

### PR DESCRIPTION
**Pull request by yasinmiran**
## Important: Merge Order
This PR **must only merge** after merging https://github.com/wso2/identity-apps/pull/1966 because it depends on `util` classes introduced in `1966`

### CODE REVIEW

File: apps/authentication-portal/src/main/webapp/consent.jsp [hunk 1]
```diff
@@ -51,6 +51,11 @@
     <% } %>
 </head>
 <body class="login-portal layout authentication-portal-layout">
+
+    <% if (new File(getServletContext().getRealPath("util/timeout.jsp")).exists()) { %>
+    <jsp:include page="util/timeout.jsp"/>
+    <% } %>
```
Comment (brionmario) [inline apps/authentication-portal/src/main/webapp/consent.jsp:57]: IMO, we should provide a WUM safe way to override/disable the timeout. @yasinmiran WDYT?

Comment (yasinmiran) [reply apps/authentication-portal/src/main/webapp/consent.jsp:57]: +1 This has been addressed in commit d48d227. Now `timeout.jsp` file refer to files in `util` directory relative to its path. Tested this and the only caveat is if the external dev wants to wrap the `timeout.jsp` in another include directive, the rendering engine will result in an infinite loop. It seems JSP doesn't like relative paths.

Comment (yasinmiran) [reply apps/authentication-portal/src/main/webapp/consent.jsp:57]: Also see changes in e8c7c49

Comment (brionmario) [reply apps/authentication-portal/src/main/webapp/consent.jsp:59]: Nice (y)

**Review by brionmario — APPROVED** [review]

**Pull request merged**